### ISSUE TITLE
why very different number for learning rate sdxl vs flux training?

### ISSUE THREAD

**Issue by thexqin** [kijai/ComfyUI-FluxTrainer#198]
this is the sample workflows from the custom node folder:

sdxl lora training:
network_dim 16
network_alpha 16
learning_rate 1.0
max_steps 3000

flux lora training:
network_dim 16
network_alpha 1.0
learning_rate 0.0004
max_steps 3000

I really appreciate this repo because it does work - my RTX5080 cannot run the flux version but sdxl lora training works fine. I'm wondering why the learning rate is so high for sdxl compared to the typical 0.00005-0.0001? Also network_alpha is not 1 as typical. Thank you.

**Comment by PumaAI487** [comment]
Kijai says on the page these are not the best settings, they are just the last ones that worked the best for him. For training advice on settings etc go to the original repo or join a discord like civit or comfys.

**Comment by thexqin** [comment]
Thank you for your response. I actually looked hard into the https://github.com/kohya-ss/sd-scripts repo but couldn't find my answers there. I thought as the author of the comfyui wrapper, the developers here may be able to offer a quick answer on just the learning_rate parameter. It is 1.0 in the sdxl lora training and it's not conventional to have a learning rate this high. I also looked into the code in this repo, specifically for sdxl lora training but limited by my ability I cannot find answer. Sorry to bother you all.

**Comment by PumaAI487** [comment]
> Thank you for your response. I actually looked hard into the https://github.com/kohya-ss/sd-scripts repo but couldn't find my answers there. I thought as the author of the comfyui wrapper, the developers here may be able to offer a quick answer on just the learning_rate parameter. It is 1.0 in the sdxl lora training and it's not conventional to have a learning rate this high. I also looked into the code in this repo, specifically for sdxl lora training but limited by my ability I cannot find answer. Sorry to bother you all.

Yeah there are a few guides on youtube, and guides on civitai if you use that site on how to use kj's trainer, i havent done any training myself but have been looking into again since i got my 3090 fixed. 

I can try to link some guides if you can't find any.

**Comment by thexqin** [comment]
If you can post any link that would be much much appreciated. There are just too many noises on the internet and many people don't actually know what they are talking about. I came here hoping to get the most accurate answer. Thank you for your time.

**Comment by PumaAI487** [comment]
https://youtu.be/d49mCFZTHsg
https://youtu.be/Altx9cxclSQ
https://youtu.be/U9QKhRMKIwc

Heres a few, and a civitai url with some guides on lora training using ai toolkit
https://civitai.com/search/articles?sortBy=articles_v5&query=ai%20toolkit

Hope some of these help

**Comment by thexqin** [comment]
Thank you a ton!
